packages/react-bootstrap-table2/src/cell.js

```diff
@@ -85,12 +93,12 @@ class Cell extends Component {
     if (cellClasses) cellAttrs.className = cellClasses;
 
     if (!_.isEmptyObject(cellStyle)) cellAttrs.style = cellStyle;
-
     if (editable && editMode !== Const.UNABLE_TO_CELL_EDIT) {
-      if (editMode === Const.CLICK_TO_CELL_EDIT) { // click to edit
-        cellAttrs.onClick = this.handleEditingCell;
-      } else { // dbclick to edit
-        cellAttrs.onDoubleClick = this.handleEditingCell;
+      const checkEditableType = _.isFunction(editable);
```

Comment:
Actually, we should tell `Cell` component if it is editable or not, instead of checking editable here. it should be handle on `Row` component(row.js)  https://github.com/react-bootstrap-table/react-bootstrap-table2/blob/develop/packages/react-bootstrap-table2/src/row.js#L29

Reply:
@AllenFang , I agree, this is not the place!